test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/Converters/IconUrlToImageCacheConverterTests.cs

```diff
@@ -66,22 +80,190 @@ public void Convert_WithLocalPath_LoadsImage()
             Assert.Equal(iconUrl, image.UriSource);
         }
 
-        [Fact(Skip="Fails on CI. Tracking issue: https://github.com/NuGet/Home/issues/2474")]
+        [Fact(Skip = "Fails on CI. Tracking issue: https://github.com/NuGet/Home/issues/2474")]
         public void Convert_WithValidImageUrl_DownloadsImage()
         {
             var iconUrl = new Uri("http://fake.com/image.png");
 
             var converter = new IconUrlToImageCacheConverter();
+#if DEBUG            
+            converter.Messages = new StringBuilder();
+#endif
 
             var image = converter.Convert(
                 iconUrl,
                 typeof(ImageSource),
                 DefaultPackageIcon,
                 Thread.CurrentThread.CurrentCulture) as BitmapImage;
 
+#if DEBUG
+            output.WriteLine(converter.Messages.ToString());
+#endif
+
             Assert.NotNull(image);
             Assert.NotSame(DefaultPackageIcon, image);
             Assert.Equal(iconUrl, image.UriSource);
         }
+
+        [Fact(Skip = "Fails on CI. Tracking issue: https://github.com/NuGet/Home/issues/2474")]
```

Comment:
This issue is closed. 

I'm not sure we should be pointing to a clsoed issue. 

Imho, either: 

1. The issue is open and will/should be fixed. 
1. This test doesn't need to exist.

Reply:
The core problem is that there's an issue for running those tests on Windows Server. I am thinking of migrating this test to a functional test.

Reply:
I suggest keeping the test but not enable it on the CI. It can only run on Windows Desktop with WPF suppport. I removed the tracking issue, since it is an external WPF issue.